packages/ingest/granule.js

```diff
@@ -108,17 +121,26 @@ class Discover {
     }).then((exists) => (exists ? false : file));
   }
 
+  /**
+   * Find new granules and format them as array of objects
+   *
+   * @param {Array} files - An array of objects with `bucket` and `name` properties
+   * that work as AWS S3 `Bucket` and `Key`
+   * @returns {Array} an array of granule objects
+   */
   async findNewGranules(files) {
     const checkFiles = files.map((f) => this.fileIsNew(f));
+
     const t = await Promise.all(checkFiles);
     const newFiles = t.filter((f) => f);
 
     // reorganize by granule
     const granules = {};
     newFiles.forEach((_f) => {
       const f = _f;
-      const granuleId = f.granuleId;
+      const { granuleId } = f;
```

Comment:
One of our recent PRs disabled the `prefer-destructuring` rule in eslint, which I'm pretty sure is why this was changed.  I'm open to discussion on it, but I think that `const granuleId = f.granuleId` is easier to read than ` const { granuleId } = f`

Reply:
Totally agree, I'm very happy to not use destructuring in cases like this.